### PR TITLE
Lazypredict

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xhochy** [comment]
CI is failing in `pip check`: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=366025&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823

**Comment by thewchan** [comment]
@xhochy It seems that this package requires scipy 1.5.4, which isn't available in CF. I checked with the scipy feedstock maintainers and adding 1.5.4 might take a lot of work that might not be worth it

**Comment by thewchan** [comment]
Upon the recommendation of the admins over at https://github.com/conda-forge/scipy-feedstock/pull/189#issuecomment-904817409 I'm going to skip `pip check`

File: recipes/lazypredict/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - numpy ==1.19.1
+    - pandas ==1.0.5
+    - pytest ==5.4.3
+    - python >=3.6
+    - pyyaml ==5.3.1
+    - scikit-learn ==0.23.1
+    - scipy >=1.5.3
+    - six ==1.15.0
+    - tqdm ==4.56.0
+    - xgboost ==1.1.1
```
Comment (dopplershift) [inline recipes/lazypredict/meta.yaml:38]: If the `xgboost` dependency were loosened this would probably pass on Windows...

**Comment by thewchan** [comment]
@dopplershift @xhochy what do yall think? Good to go?

File: recipes/numba_quaternion/meta.yaml [hunk 2]
```diff
@@ -1,44 +0,0 @@
-{% set name = "numba_quaternion" %}
```
Comment (dopplershift) [inline recipes/numba_quaternion/meta.yaml:1]: Shouldn't be modifying files outside your recipe...

Comment (thewchan) [reply recipes/numba_quaternion/meta.yaml:1]: Oh weird? Must be a botched merge conflict, I'll see if I can rebase that away

Comment (thewchan) [reply recipes/numba_quaternion/meta.yaml]: @dopplershift how about now?

Comment (dopplershift) [reply recipes/numba_quaternion/meta.yaml]: 👍

**Review by dopplershift — APPROVED** [review]
Pending CI

File: recipes/numba_quaternion/meta.yaml (hunk 2 above)
Comment (thewchan) [reply recipes/numba_quaternion/meta.yaml]: Thanks for all your help @dopplershift, I wouldn't know what to do otherwise!

**Comment by thewchan** [comment]
@conda-forge/staged-recipes Looks like everything pass and it's ready for any final reviews. Thanks!

**Pull request merged**